components/filter/index.jsx

```diff
@@ -239,7 +239,7 @@ class Filter extends React.Component {
 				)}
 			>
 				{!this.props.isLocked && (this.props.children || this.props.popover) ? (
-					<Popover {...popoverProps}>
+					<Popover {...popoverProps} silenceDeprecatedPropertyWarning>
```

Comment:
This is internal. Why is there a deprecated prop in use? Should the deprecated prop be removed from an example?

Reply:
I made sure the `silence{WarningType}` stuff was never included in any examples shown on the site. In nearly all cases, our storybook stories or browser tests are still testing the deprecated features or components, hence the need for silencing. This is the only exception and I wasn't quite sure what to do with it... check this out:

```
const defaultPopoverProps = {
	ariaLabelledby: `${this.getId()}-popover-heading`,
	align: this.props.align,
	body: popoverBody,
	heading: '',
	id: this.getId(),
	isOpen: this.state.popoverIsOpen,
	// MAGIC NUMBERS - REMOVE/REDESIGN WHEN DESIGN FOR RIGHT-ALIGNED FILTERS ARE ADDED TO SLDS
	offset: this.props.align === 'right' ? '0px -35px' : undefined,
	onClose: this.handleClose,
	onRequestClose: this.handleClose,
	position: 'overflowBoundaryElement',
	triggerClassName: 'slds-grow',
};
```

The `offset` value has a note on it in regards to updating this. @interactivellama, are you aware if that has happened yet? Either way, `offset` seems critically important to the functionality here for the time being. I figured disabling the warning was acceptable since our own component is using it (hence it wouldn't be seen by developers) and we could create a separate issue to figure out how to make this work better without offset later.